Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM mhart/alpine-node
+FROM alpine:edge
 
 WORKDIR /usr/src/app
 COPY Gemfile /usr/src/app/
 COPY Gemfile.lock /usr/src/app/
 
-RUN npm install -g csslint && \
-    apk --update add ruby ruby-dev ruby-bundler less ruby-nokogiri build-base && \
+RUN apk --update add nodejs git ruby ruby-dev ruby-bundler less ruby-nokogiri build-base && \
+    npm install -g codeclimate/csslint.git#2a53712c61710840c023978418c7b48e3f32ac64 && \
```

Comment:
It was originally on the first line of the `RUN` command. I had to move it to the second because it require git now.